src/lib/fb.js

```diff
@@ -1,36 +1,33 @@
 import {firebaseConfig} from 'webdev_config';
 import {store} from './store';
 import {clearSignedInState} from './actions';
-import firestoreLoader from './firestore-loader';
+import firebaseLoader from './utils/firebase-loader';
 
-/* eslint-disable require-jsdoc */
-
-firebase.initializeApp(firebaseConfig);
-
-// Initialize performance monitoring
-firebase.performance();
+const firebasePromise = firebaseLoader('app', 'auth', 'performance');
```

Comment:
I mean rename `const firebasePromise` to `const loadFirebase` but maybe a better name would be `const isFirebaseLoaded` so we could do `isFirebaseLoaded.then(...`

Reply:
It may also be worth renaming `firebaseLoader` to `queue()` or something. At first glance `firebaseLoader` seems like it's going to load something, but it's just a function that _gives you_ a loader which you still have to call yourself.